modules/motions/PlaCameraAction.cpp

```diff
@@ -0,0 +1,105 @@
+/**
+ * @file   PlaCameraAction.cpp
+ * @brief  動体検知用動作クラス
+ * @author miyahara046 HaruArima08
+ */
+#include "PlaCameraAction.h"
+#include <thread>
+
+PlaCameraAction::PlaCameraAction(Robot& _robot, double _threshold, double _minArea,
+                                 const cv::Rect& _roi)
+  : Motion(_robot), motionDetector(_threshold, _minArea, _roi)
+{
+}
+
+void PlaCameraAction::getBackgroundFrame()
+{
+  cv::Mat frame;
+  while(1) {
+    // 背景に適しているか判断するフレームの獲得
+    for(int counter = 0; counter < 5; counter++) {
+      robot.getCameraCaptureInstance().getFrame(frame);
+      std::this_thread::sleep_for(std::chrono::milliseconds(33));
+    }
```

Comment:
これは、多分最初の撮影時にフレームを取得できない問題の対応だと思うんだけど、実際にこの動作つかうときは、もう既に何枚も画像取得した後だと思うから、いらない気がするかなー

```suggestion
    robot.getCameraCaptureInstance().getFrame(frame);
```